lib/Shipment/UPS.pm

```diff
@@ -600,73 +555,29 @@ sub _build_services {
     }
 
     $self->notice( '' );
-    if ( $response->get_Response->get_Alert ) {
-      foreach my $alert (@{$response->get_Response->get_Alert}) {
-        warn "Notice: " . $alert->get_Description->get_value if $self->debug;
-        $self->add_notice( $alert->get_Description->get_value . "\n" );
+    if ( $response->{Response}->{Alert} ) {
+      foreach my $alert (@{$response->{Response}->{Alert}}) {
+        warn "Notice: " . $alert->{Description} if $self->debug;
+        $self->add_notice( $alert->{Description} . "\n" );
       }
     }
 
   } catch {
-      warn $_ if $self->debug;
-      try {
-        my $error = join "\n",  map {
-          $_->get_PrimaryErrorCode()->get_Code() . ' - ' . $_->get_PrimaryErrorCode()->get_Description;
-        } @{ $response->get_detail()->get_Errors()->get_ErrorDetail() };
-        warn $error if $self->debug;
-        $self->error( $error );
-      } catch {
-        warn $_ if $self->debug;
-        warn "Error: " . $response->get_faultstring if $self->debug;
-        $self->error( $response->get_faultstring->get_value );
-      };
+    my $error = $_;
+    warn $error if $self->debug;
+    $self->error( $error );
   };
-  }
 
   if ($self->surepost) {
     if ($self->error) {
       $self->add_notice( 'All services other than SurePost failed due to error: ' . $self->error . "\n" );
       $self->error('');
     }
-
-    if ( $self->weight_unit eq 'oz' ) {
-        $services{92} = Shipment::Service->new(
-            id   => '92',
-            name => $self->service_map->{92},
-        );
-
-        $services{surepost_oz} = $services{92};
```

Comment:
Support for SurePost service 92 has also been removed, can this be added back in or was there a reason it was removed? See commit b56dba8c2fbb